wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -291,16 +291,58 @@ $( document ).ready(function() {
     jQuery(document).on('click', '#sortreverse', function(e) {
         e.preventDefault()
 
-        jQuery('.arcsi-blokks').html(jQuery('.arcsi-blokks').find(jQuery('.arcsi-blokks .arcsiblokk')).get().reverse());
+        const mylist = $('.arcsi-blokks');
+        let listitems = mylist.children(".arcsiblokk");
 
         if (ascending) {
             ascending = false
             $(this).find('i').removeClass('fa-sort-numeric-asc').addClass('fa-sort-numeric-desc')
+            listitems.sort(function(b, a) {
+            let compA = $(a).find('.airtime').text().toUpperCase();
+            let compB = $(b).find('.airtime').text().toUpperCase();
+                return (compA < compB) ? -1 : (compA > compB) ? 1 : 0;
+            })
+            $(mylist).append(listitems);  
         } else {
             ascending = true
             $(this).find('i').removeClass('fa-sort-numeric-desc').addClass('fa-sort-numeric-asc')
+             listitems.sort(function(a, b) {
+            let compA = $(a).find('.airtime').text().toUpperCase();
+            let compB = $(b).find('.airtime').text().toUpperCase();
+                return (compA < compB) ? -1 : (compA > compB) ? 1 : 0;
+            })
+            $(mylist).append(listitems);           
         }
     })
 
+    let alphabetascending = true
+    jQuery(document).on('click', '#alphabetical', function(e) {
+        e.preventDefault()
+
+        const mylist = $('.arcsi-blokks');
+        let listitems = mylist.children(".arcsiblokk");
+
+        if (alphabetascending) {
+            alphabetascending = false
+            $(this).find('i').removeClass('fa-sort-alpha-asc').addClass('fa-sort-alpha-desc')
+            listitems.sort(function(b, a) {
+            let compA = $(a).find('h4').text().toUpperCase();
```

Comment:
I see that currently the name is in a `h4` element but it makes the code hard to read (vs `airtime` is much better in this regard). We could adjust the PHP template to generate better readable HTML code.

Reply:
whatever, yeah, it could be more readable, sorry for not thinking about it